ZenPacks/zenoss/LinuxMonitor/PhysicalVolume.py

```diff
@@ -34,3 +34,8 @@ def harddisk(self):
         for harddisk in self.device().hw.harddisks():
             if harddisk.id == self.harddisk_id:
                 return harddisk
+
+    def getRRDTemplates(self):
+        templates = super(PhysicalVolume, self).getRRDTemplates()
+        templates.append(self.getRRDTemplateByName('IOStat'))
```

Comment:
As noted above, I don't think you should have an IOStat monitoring template that's applied both the HardDisk and PhysicalVolume.

As a side note, the list returned by getRRDTemplates must only include RRDTemplate objects. This code would append a `None` to the returned list of for some reason there was no IOStat monitoring template. That would break things.